src/main/java/app/persistence/OrderMapper.java

```diff
@@ -92,6 +92,34 @@ public List<Order> getAllCustomerOrders(int customerId, ConnectionPool connectio
         return null;
     }
 
+    public static Order getOrderById(int orderId, ConnectionPool connectionPool) throws DatabaseException {
+        String sql = "SELECT account_id, order_title, carport_width, carport_length, order_status, order_total_price, order_timestamp  FROM orders WHERE account_id=?";
```

Comment:
The current sql code is looking for an order where `account_id = ?` but the method should be looking for an order with a specific id and not look for one with a specific account id